examples/mock-data-reader/config_file.json

```diff
@@ -0,0 +1,26 @@
+{
+  "/ENTRY/title": "@eln",
+  "/ENTRY/USER[user]": {
+    "name":"@eln",
+    "address":"@eln"
+  },
+  "/ENTRY/INSTRUMENT[instrument]": {
+    "@version":"@attrs:metadata/instrument/version",
+    "DETECTOR[detector]":{
+      "count_time":"@attrs:metadata/instrument/detector/count_time",
+      "count_time/@units":"@attrs:metadata/instrument/detector/count_time_units"
+    }
+  },
+  "/ENTRY/SAMPLE[sample]": {
+    "name":"@eln",
+    "physical_form":"@eln",
+    "temperature":"@eln",
+    "temperature/@units":"@eln"
```

Comment:
We can really only either do `@eln` using the `CONVERT_DICT` and the config key OR the `@eln:path/in/dict` notation. As outlined above, I opted for the first option because the second one is covered by the `@attrs` examples.

I did not use the `!` here in order not to overcomplicate things. The user can just check the MultiReaderFormat docs in `learn` for these special use cases.

Reply:
You could use both, if you check for the value of path. But maybe that is not good practice anyhow. I suggest then to keep the examples as they are, but mention there is also the other way for eln entries.

Reply:
Actually, we can just still use `@eln:"/ENTRY[entry]/USER[user]/address"` and change the `get_eln_data` to use either the key or the path. Will do.

Reply:
Sorry, didn't read your previous comment. I agree it's probably best to not mix these. I'll just say that you can also not do any replacement and use the path 👍